MainApp/AssemblyOne/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace AssemblyOne
+{
+    public class AssemblyOneType
+    {
+        private int privateInt = 1;
```

Comment:
in future, for private fields use please the following declaration:

`private int privateInt = 1;`  => `private int _privateInt = 1;`